### ISSUE TITLE
As a data archivist, I want to use day^-1 as a permissible value for unit

### ISSUE THREAD

**Issue by murashinln** [NASA-PDS/pds4-information-model#660]
### Checked for duplicates

No - I haven't checked

### 💪 Motivation

...so that I can describe a sensitivity degradation rate for detector of an imager in the label.

### 📖 Additional Details

I'm preparing the PDS4 bundle for the Longwave Infrared Camera instrument (LIR) onboard the Venus Climate Orbiter Akatsuki spacecraft. I defined vco:lir_sensitivity_degradation_rate attribute for LIR calibrated data in VCO LDD, and value of the attribute has a unit of day^-1. It seems to me that Units_of_Rates is an appropriate class to define it, but it's ok defining it in other class if you think it is more suitable.

### Acceptance Criteria

**Given** 
**When I perform** 
**Then I expect**

**Comment by cgobat** [comment]
Seems like `Units_of_Frequency` might make sense for this, since that's where all the other inverse time units currently live. A stopgap measure could even be to multiply your value(s) by ~0.01157 and use the 'mHz' (i.e. $10^{-3}\ \text{s}^{-1}$) unit that already exists.

**Comment by matthewtiscareno** [comment]
The difference between `Units_of_Rates` and `Units_of_Frequency` appears to be a subtle one.  A rate should specify what it is that is happening per unit time (the only current examples are counts/bin and kilobits/s).  A frequency is understood to be some kind of cycle per unit time. 

Both of these have units of one over time.  For that matter `Units_of_Angular_Velocity` also has units of one over time. 

@murashinln, can you be more specific about how exactly you are measuring sensitivity degradation?  Is there some specific quantity whose rate you are reporting?

**Comment by murashinln** [comment]
Sensitivity degradation is estimated from measured radiance for deep space pixel which is decreasing in time. The sensitivity degradation is modeled as a linear function of time ([Taguchi+2023](https://doi.org/10.1186/s40623-023-01803-w)), as described below:

A sensitivity degradation rate $\alpha$ is used in an equation for sensitivity degradation correction, $I_o(T, t) = (1+\alpha t')\\{I(T) - I(T_s)\\} + I(T_s)$, where $I_o(T, t)$ is an observed radiance for a target temperature $T$ at an observation time $t$, $t'$ is an operating time of the instrument at the observation time $t$, $I(T)$ is a corrected radiance for a target temperature $T$, and $I(T_s)$ is a radiance for a shutter temperature $T_s$.

**Comment by matthewtiscareno** [comment]
@murashinln:  Okay, so it seems like α is the rate at which a dimensionless quantity is decreasing linearly with time.  Correct? 

In that case, perhaps `Units_of_Rates` makes the most sense, as you suggested in the OP.  @cgobat, would you agree?

**Comment by murashinln** [comment]
@matthewtiscareno Correct. To be clear, we are thinking on radiance, but a sensitivity degradation is represented as a part of scaling factor $1+\alpha t'$. If there is an image with 20% sensitivity degradation, then $\alpha t' = -0.2$ and $1+\alpha t' = 0.8$.

**Comment by jordanpadams** [comment]
@murashinln @murashinln @matthewtiscareno do we have agreement that this will require an update to the PDS4 Standard? If so, we can get a ticket created with the DDWG to review this request.

**Comment by murashinln** [comment]
@jordanpadams I think we have agreement. @cgobat reacted with thumbs up emoji to https://github.com/NASA-PDS/pds4-information-model/issues/660#issuecomment-1681732496 .

**Comment by jordanpadams** [comment]
@rsjoyner would you mind creating an SCR for this with the DDWG?

**Comment by murashinln** [comment]
@jordanpadams hi, how is the current status?

**Comment by murashinln** [comment]
@matthewtiscareno Thanks. I don't have access to it.

**Comment by jshughes** [comment]
@jordanpadams PR #1022 resolves this ticket. - Add enumerated value day**-1 to Units_of_Rates#1022

Would you prefer that the PR be updated to include a reference to this ticket, after the fact?

**Comment by jordanpadams** [comment]
duplicate of https://github.com/NASA-PDS/pds4-information-model/issues/1004

**Comment by jordanpadams** [comment]
@jshughes actually this looks like a duplicate of the SCR that written. so closing this as a duplicate versus closed as completed. https://github.com/NASA-PDS/pds4-information-model/issues/1004

to do that in the future, you can just reference the duplicate issue, close the issue, and add the `duplicate` label above.
